### ISSUE TITLE
Blank White Screen - MacOS High Sierra

### ISSUE THREAD

**Issue by lachie-92** [desktop/desktop#4833]
## Description

Since this morning, cannot open github desktop, once opened, it says it is running, but only shows a white blank page. Uninstalled github desktop and installed again did not do anything.

## Version

* GitHub Desktop: 1.20 then 1.22(download to reinstall)

* Operating system: Version 10.13.4

### Expected Behavior

Open Github Desktop

### Actual Behavior

Blank White Screen

### Logs

Log files seem to be deleted since i uninstalled github desktop?

**Comment by lachie-92** [comment]
@shiftkey thanks i have just done that, its given me a "PrematureCommitError" and pointing me to the following webpage: http://bit.ly/2kdckMn 

I am trying to find what has been commited early?

**Comment by shiftkey** [comment]
This is a duplicate of #4054, but the workaround mentioned here https://github.com/desktop/desktop/issues/4054#issuecomment-367567297 was for Windows. I'm going to cross-reference this thread there and add a workaround for macOS that should unblock you.
